tests/components/mqtt/test_cover.py

```diff
@@ -260,8 +260,8 @@ async def test_state_via_template(hass, mqtt_mock):
     assert state.state == STATE_CLOSED
 
 
-async def test_state_via_template_with_json_value(hass, mqtt_mock, caplog):
-    """Test the controlling state via topic with JSON value."""
```

Comment:
It's easier to understand the impact of the change if you don't remove tests, but only add new ones, can you re-add the ones which are removed?

Reply:
I have validated that all previous tests pass so indeed we should keep them, it makes sure there is no breaking change in this PR

Reply:
agree, was not intentional